flutter_gallery_app/lib/feature/home/wrap_screen/widgets/wrap_screen_config.dart

```diff
@@ -0,0 +1,151 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:flutter_gallery_app/constants/constants.dart';
+import 'package:flutter_gallery_app/feature/home/wrap_screen/cubit/wrap_screen_cubit.dart';
+import 'package:flutter_gallery_app/widgets/option_item.dart';
+import 'package:flutter_gallery_app/widgets/switch_item.dart';
+
+class WrapScreenConfig extends StatelessWidget {
+  const WrapScreenConfig({
+    super.key,
+  });
+
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+      padding: const EdgeInsets.symmetric(horizontal: 15, vertical: 10),
+      decoration: BoxDecoration(
+        color: Colors.white,
+        boxShadow: [
+          BoxShadow(
+            color: Colors.grey.withOpacity(0.5),
+            spreadRadius: 5,
+            blurRadius: 7,
+            offset: const Offset(0, 5),
+          ),
+        ],
+      ),
+      child: BlocBuilder<WrapScreenCubit, WrapScreenState>(
```

Comment:
Tách ra rồi thì BlocBuilder ở đây ko cần nữa nha em, để ở đây thì vẫn sẽ build lại toàn bộ khi có bất kì state nào thay đổi nha